response_operations_ui/controllers/uaa_controller.py

```diff
@@ -227,6 +227,9 @@ def update_user_account(payload):
         if response.status_code == 404:
             # User id not found
             errors = {"user_id": ["User id not found"]}
+        elif response.status_code == 400:
+            # Username already exists
+            errors = {"response": [400, "Username already in use. Please try another"]}
```

Comment:
For this, why can't we just use the format that the other ones use? It could look something like:
```python
errors = {"status_code": response.status_code, "message": "Username already in use. Please try another"}
```